aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -282,6 +282,11 @@ at::Tensor PackedConvWeight<kSpatialDim>::apply_impl(
   ConvDimChecks<kSpatialDim>(
       act.ndimension(), stride().size(), padding().size(),
       output_padding().size(), dilation().size(), func_name, transpose());
+  // TODO(before land): figure out if this check needs to be architecture
```

Comment:
I think safe_on_fbgemm is not strictly correct. It is more fbgemm for skylake (T1) machines. We should do the run-time check based on the architecture info also.

Reply:
@dskhudia , would you have any code example for the right check here you can point me to?

Reply:
note, I'm going to move this to the next PR before landing, to decouple storing the flag from using the flag